terraform/eks/daemon/gpu/main.tf

```diff
@@ -701,19 +701,24 @@ resource "kubernetes_cluster_role_binding" "rolebinding" {
     namespace = "amazon-cloudwatch"
   }
 }
-
 resource "null_resource" "validator" {
   depends_on = [
     aws_eks_node_group.this,
     kubernetes_daemonset.service,
     kubernetes_cluster_role_binding.rolebinding,
     kubernetes_service_account.cwagentservice,
   ]
+
   provisioner "local-exec" {
     command = <<-EOT
-      echo "Validating EKS metrics/logs for EMF"
       cd ../../../..
-      go test ${var.test_dir} -eksClusterName=${aws_eks_cluster.this.name} -computeType=EKS -v -eksDeploymentStrategy=DAEMON -eksGpuType=nvidia
+      i=0
+      while [ $i -lt 10 ]; do
+        i=$((i+1))
+        go test ${var.test_dir} -eksClusterName=${aws_eks_cluster.this.name} -computeType=EKS -v -eksDeploymentStrategy=DAEMON -eksGpuType=nvidia && exit 0
+        sleep 60
+      done
+      exit 1
```

Comment:
Why are we changing the daemon/gpu test?

Reply:
It basically needs more time to load into the aws console. I'm just adding more retries. Currently the test fails once and then deletes terraform/runs it again, this instead reduces the process to just rerunning the test.

Reply:
This is unrelated to the test you're adding though right?

Reply:
This was added in order to make sure the test does not break.